### DISCUSSION TITLE
Privacyguides instead of privacytools

### DISCUSSION THREAD

**Discussion by jermanuts** [simeononsecurity/Windows-Optimize-Harden-Debloat#35] [General]
Mind changing privacytools.io in README.md to privacyguides.org (quick summary the whole team behind privacytools is not privacyguides, more details in the link below)

https://www.privacyguides.org/blog/2021/09/14/welcome-to-privacy-guides/

**Comment by simeononsecurity** [comment]
The blog doesn't go into how why and who controls the privacytools.io domain or how they "lost" access. 
Additionally, privacytools.io seems to actually be still updated independently.  In fact, it appears to be updated more often than the privacyguides.org website.  
When we linked privacytools.io, we linked it because it was part of the inspiration and was responsible for part of the ideas, at least indirectly, for the automations we implemented in our scripts.
We'll take this under consideration to add the second resource. However something just doesn't seem right here. It's unclear the correct move forward.
Thanks for bringing this to our attention.

**Reply by jermanuts** [reply]
It's controlled by the original founder of privacytools.io who is incompetent and his recommendations are affiliate-based like nordvpn and many others.. This blog explains in depths based on his recommendations https://deploy-preview-35--privsec-dev.netlify.app/knowledge/privacytools.io/  

privacyguides is the whole team that decided to start their own project because they were the ones that actually does research before recommending software or a service.
